modules/ROOT/pages/platform/security/single-sign-on.adoc

```diff
@@ -5,49 +5,57 @@
 label:AuraDB-Enterprise[]
 label:AuraDS-Enterprise[]
 
-Aura Enterprise supports Single Sign-On (SSO) at both the Console level and for accessing Workspace, Bloom and Browser clients directly at the instance level.
+== SSO levels
+
+Aura admins can configure organization and tenant level SSO:
+
+* *Organization level SSO:* Applies to everything related to the console within an organization. Grants console access for organization management, this includes editing organization and tenant settings, viewing the list of instances within tenants, and managing instances (such as pause/resume). However, it does not grant access to view or modify data within the database instances themselves. Currently, role mapping is not applicable to Organization level SSO.
 
-[NOTE]
-====
-Accessing Aura with SSO requires:
- 
-* Authorization Code Flow with PKCE.
-* A publicly accessible Identity Provider (IdP) server.
-====
 
-== Console SSO
+* *Instance level SSO:*  Affects all new database instances created within that tenant. It ensures users logging in with SSO have access to the database instances within the tenant. Instance level SSO gives access to to view or modify data within the database instances themselves. For this level, the role mapping may be used to grant users different levels of access based on a role in their IdP (it *does not* give access to edit the tenant settings e.g. edit the tenant name / network access, or edit the instance settings e.g. rename an instance / pause and resume).
```

Comment:
> Instance level SSO gives access to to view or modify data within the database instances themselves

This is not true, it depends on the role mapping if they can actually view/modify data in the instance.